HACKING.rst

```diff
@@ -28,6 +31,10 @@ you may need to run::
 
   pip install importlib
 
+You may get an error 'psycopg2 package not found' when you do 'haas init_db' 
+in the next step if you are using PostgreSQL database. You may need to run::
+
+  pip install psycopg2
```

Comment:
Thing is, if in the future we add sort for more databases (e.g. mysql), this will become "install the library for your database," which won't necessarily be the same for everyone.